tools/cancellation.f90

```diff
@@ -0,0 +1,6 @@
+program cancellation
+      implicit none
+
+      print*, "here we go!"
```

Comment:
Ok, michel, this looks good - but there must be more to this no? And where is the documentation?!